packages/loopring_v3/contracts/amm/libamm/AmmData.sol

```diff
@@ -83,6 +103,9 @@ library AmmData
 
         Token[]  tokens;
         uint96[] tokenBalancesL2;
+        uint96[] vTokenBalancesL2;
```

Comment:
tokenL2Balances
tokenL2VirtualBalances or tokenL2VBalances

Reply:
Because balances also include L1 balances now, what do you think of just `balances` and `vBalances`?